api/voice-react-native-sdk.api.md

```diff
@@ -544,6 +601,48 @@ namespace MediaErrors {
     }
 }
 
+// @public
+export interface OutgoingCallMessage {
+    addListener(failureEvent: OutgoingCallMessage.Event.Failure, listener: OutgoingCallMessage.Listener.Failure): this;
+    addListener(sentEvent: OutgoingCallMessage.Event.Sent, listener: OutgoingCallMessage.Listener.Sent): this;
+    addListener(outgoingCallMessageEvent: OutgoingCallMessage.Event, listener: OutgoingCallMessage.Listener.Generic): this;
+    // @internal (undocumented)
+    emit(failureEvent: OutgoingCallMessage.Event.Failure, error: TwilioError): boolean;
+    // @internal (undocumented)
+    emit(sentEvent: OutgoingCallMessage.Event.Sent): boolean;
+    // @internal (undocumented)
+    emit(outgoingCallMessageEvent: OutgoingCallMessage.Event, ...args: any[]): boolean;
+    on(failureEvent: OutgoingCallMessage.Event.Failure, listener: OutgoingCallMessage.Listener.Failure): this;
+    on(sentEvent: OutgoingCallMessage.Event.Sent, listener: OutgoingCallMessage.Listener.Sent): this;
+    on(outgoingCallMessageEvent: OutgoingCallMessage.Event, listener: OutgoingCallMessage.Listener.Generic): this;
+}
+
+// @public
+export class OutgoingCallMessage extends EventEmitter implements CallMessage {
+    constructor({ content, contentType, messageType, voiceEventSid, }: NativeCallMessageInfo);
+    _content: string;
+    _contentType: CallMessage.ContentType;
+    getContent(): string;
+    getContentType(): CallMessage.ContentType;
+    getMessageType(): CallMessage.MessageType;
+    getSid(): string | undefined;
+    _messageType: CallMessage.MessageType;
+    _voiceEventSid?: string;

```

Comment:
These members are public, because I have `OutgoingCallMessage` implement `CallMessage`, if I make them private I get type error.

https://github.com/twilio/twilio-voice-react-native/blob/VBLOCKS-2464/feat-call-message/src/OutgoingCallMessage.ts#L154